client/material/button.tsx

```diff
@@ -28,26 +28,42 @@ export const Label = styled.span`
   white-space: nowrap;
 `
 
-export interface HotkeyProp {
-  keyCode: number
-  altKey?: boolean
-  shiftKey?: boolean
-  ctrlKey?: boolean
+export function useButtonRef(
```

Comment:
I find this function pretty strange to be here and to be named this way. It really has nothing to do with buttons at all, it's just a multiplexing ref? If you want that, I guess fine, but it should probably live elsewhere and be named as such (and take a generic param instead of being hardcoded to `HTMLButtonElement`). If you're having trouble with a name, I'd probably do `useMultiRef`